### ISSUE TITLE
does not ask for permission.

### ISSUE THREAD

**Issue by nihal697** [vhaudiquet/BladePlayer#30]
Not a major issue.
While adding local storage in the sources, tapping on "Grant permission..." does not do anything. It should take you to the all files access page.
Thanks

**Comment by davidsantana980** [comment]
How is this not a major issue? Is there a workaround I don't know about?

**Comment by nihal697** [comment]
> How is this not a major issue? Is there a workaround I don't know about?

Yes you can grant the permission from settings > apps > special app access > all files access.
This is for pixel like roms or close to aosp roms. Your devices rom might have it somewhere else you can search for.
